README.md

```diff
@@ -0,0 +1,54 @@
+# cr-logging
+A wrapper for initialising Serilog consistently.
+
+## Configuration
+
+### Parameters
+
+All the appsettings are not mandatory; the program will not fail of these are not specified in the app.config, or their values are empty/whitepace.
```

Comment:
We should probably rewrite this sentence, `not mandatory` -> `optional` for example. Rather than the program will not fail we should just say it will default to the values specified, etc.